app/services/validators/common-validator.js

```diff
@@ -21,6 +21,10 @@ exports.isGreaterThanZero = function (value) {
   return value > 0
 }
 
+exports.isGreaterThanOrEqualToZero = function (value) {
+  return value >= 0
```

Comment:
Just a general comment about this module - we have a check on the method below for value !== null, but not on the other functions.

Also, the function below for isLessThanMaximumDifferentApprovedAmount looks like it will fail if the value is null as the check happens after the && i.e. the value was actually null then the second half of the && would never execute.